Calvin-TMO/TMO
# More emails

To test:
1. Make yourself a professor in the database (you can assign yourself a role using tmo_admin)
2. Make a new user who is a tutor with an email address that you can access
3. Make a new assignment where the tutor whom you have just created tutors for your class where you are the professor. The tutee may be anyone.
4. To test whether emails are being sent when a tutor submits a new report:
Log in to your tutor account and make a new report. Submit the report and check your professor email address
5. To test whether emails are being sent when someone submits a comment to a report:
Log in to your account that is a professor and submit a comment on the report created in the step above. Check your tutor account's email.

Hopefully you like the design. Feel free to give any feedback.
